### PR TITLE
[fix](load) load core dump print load id

### PR DESCRIPTION
**Pull request by xinyiZzz**
## Proposed changes

save the load id to the thread context,
expect all task ids to be saved in thread context, compaction/schema change/etc.
image

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xinyiZzz** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.99 seconds
 stream load tsv:          504 seconds loaded 74807831229 Bytes, about 141 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.0 seconds inserted 10000000 Rows, about 344K ops/s
 storage size: 17162023501 Bytes

**Review by dataroaring — DISMISSED** [review]
LGTM

File: be/src/service/internal_service.cpp [hunk 1]
```diff
@@ -373,6 +377,10 @@ void PInternalServiceImpl::_tablet_writer_add_block(google::protobuf::RpcControl
         int64_t execution_time_ns = 0;
         {
             SCOPED_RAW_TIMER(&execution_time_ns);
+            TUniqueId tid;
+            tid.__set_hi(request->id().hi());
+            tid.__set_lo(request->id().lo());
+            SCOPED_ATTACH_TASK(tid);
```
Comment (yiguolei) [inline be/src/service/internal_service.cpp:383]: 这样做可能不对，因为导入的内存是手动统计的，如果在一个be 上有insert into select，如果这里自动统计出错，那么可能导致select 端被挂掉。

**Comment by xinyiZzz** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 47.06 seconds
 stream load tsv:          503 seconds loaded 74807831229 Bytes, about 141 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          67 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17162588514 Bytes

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**